examples/imsim-user.yaml

```diff
@@ -0,0 +1,20 @@
+# Use imSim custom modules
+modules:
+    - imsim
+
+# Get most of the configuration from the imSim named template
+template: imsim-config
+
+input.instance_catalog.file_name: $os.environ.get('IMSIM_HOME')+'/imSim/examples/example_instance_catalog.txt'
+input.instance_catalog.sed_dir: $os.environ.get('SIMS_SED_LIBRARY_DIR')
```

Comment:
We probably don't want two versions of this file.  One in examples and one in config.  I think here is the better spot, so should probably remove the one in the config directory.

Reply:
Yes, I agree.. the config should only be for config files.  I will remove.